react/src/mocks/functions.ts

```diff
@@ -16,3 +18,23 @@ export function mockMatchMediaQuery(returnBool: Boolean) {
     })),
   });
 }
+
+// mock an Apollo GraphQLError
+export const mockGraphQLError = (query, variables = {}) => ({
```

Comment:
:+1: for removing duplication in the tests with these functions. Also @vahidbazzaz for further use